go.mod

```diff
@@ -37,6 +37,6 @@ require (
 	k8s.io/kubectl v0.21.0-beta.1
 	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
 	sigs.k8s.io/controller-runtime v0.9.0-alpha.1
-	sigs.k8s.io/kind v0.9.0
+	sigs.k8s.io/kind v0.10.1-0.20210406115533-b6bc11252265
```

Comment:
Was 0.11 not tagged yet?

Reply:
Afaik current plan is to release & tag next week. I'm mostly testing this to see if it would fix our failed upgrade test.

Reply:
But overall we have to decide sooner or later if it makes sense to use latest kind version when we try to test against latest Kubernetes. 

I think based on 1.21 (as an example) discussed here: https://github.com/kubernetes-sigs/kind/issues/2189#issuecomment-816313206 it could make sense to stay on the main-branch of kind to get the latest fixes for the latest Kubernetes releases. I guess otherwise we will always hit issues because the last kind release does not support the last Kubernetes release or the last changes on Kubernetes main.